autoemulate/core/compare.py

```diff
@@ -451,6 +452,13 @@ def plot(  # noqa: PLR0912, PLR0915
             The index of the input feature to plot against the output.
         output_index: int
             The index of the output feature to plot against the input.
+        figsize: tuple[int, int] | None
+            The size of the figure to create. If None, it is set based on the number
+            of input and output features.
+        ncols: int
```

Comment:
```suggestion
        n_cols: int
```

Reply:
Just to note if this is updated to `ncols` https://github.com/alan-turing-institute/autoemulate/pull/734/commits/0f616ad8fa72a3a1e2275cd259aa4fbfc374842c#r2277070034 then this suggestion is obsolete.